scripts/deployment/staking/README.md

```diff
@@ -2,16 +2,15 @@
 
 ## Arbitrum
 1. On L1: deploy ArbitrumDepositProcessorL1
-2. On L2: Calculate L2 alias for the ArbitrumDepositProcessorL1.address
-3. On L2: Deploy ArbitrumTargetDispenserL2
-4. On L1: In ArbitrumDepositProcessorL1 set ArbitrumTargetDispenserL2 to l2TargetDispenser.
+2. On L2: Deploy ArbitrumTargetDispenserL2
+3. On L1: In ArbitrumDepositProcessorL1 set ArbitrumTargetDispenserL2 to l2TargetDispenser.
 
 ## Gnosis
 1. On L1: deploy GnosisDepositProcessorL1
 2. On L2: deploy GnosisTargetDispenserL2
 3. On L1: In GnosisDepositProcessorL1 set GnosisTargetDispenserL2 to l2TargetDispenser.
 
-## Optimism / Base
+## Optimism / Base / Mode
```

Comment:
in the future we will have also celo here, right?

Reply:
Correct, can't do it for now as OLAS is bridged via the Wormhole.